src/Menu/README.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Menu'
+type: 'component'
+components:
+- MenuItem
+- Menu
+categories:
+- Navigation
+- Buttonlike
+status: 'New'
+designStatus: 'Done'
+devStatus: 'Done'
+notes: ''
+---
+
+### MenuItem
+
+A menu item is a link, button, or checkbox for use by any kind of menu overlay, dropdown menu, or nav menu. A menu item can be text-only or combined with an icon.
```

Comment:
I attempted to keep this as short and informative as possible. Should I mention the "as" prop more in order to spell out plainly how to use it or is the example fine?